posthog/hogql/property.py

```diff
@@ -881,6 +881,54 @@ def selector_to_expr(selector_string: str):
     return ast.And(exprs=exprs)
 
 
+def filters_to_expr(filters: dict, team: Team) -> ast.Expr:
```

Comment:
Would prefer this to not go into the hogql lib with a generic `dict` arg - can this be typed? It looks like we're expecting certain keys, lets be safe and use good typing